### PR TITLE
Issue: Add theme

### PR DESCRIPTION
**Pull request by madhusudhand**
#### Changes proposed in this Pull Request:

This adds a new theme `Issue` created by @henriqueiamarino

image

Note: Check the availability of the name `issue` and rename the theme.

### CODE REVIEW

File: theissue/parts/post-meta.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"1rem"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"0px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:paragraph {"fontFamily":"font-awesome-6-free-solid"} -->
+<p class="has-font-awesome-6-free-solid-font-family"></p>
```
Comment (madhusudhand) [inline theissue/parts/post-meta.html]: Can you confirm if this is the intended character?

image

File: theissue/templates/page.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"4rem","left":"4rem"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"0px"}},"layout":{"type":"default"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0px;margin-bottom:0px;padding-right:4rem;padding-left:4rem"><!-- wp:template-part {"slug":"header-pages"} /-->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"0rem","right":"0rem","bottom":"0rem","left":"0rem"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"0px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0rem;padding-right:0rem;padding-bottom:0rem;padding-left:0rem"><!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"0px","bottom":"0px","right":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"0px"}}} -->
+<div class="wp-block-group alignfull" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:post-content /--></div>
+<!-- /wp:group --></div>
```
Comment (madhusudhand) [inline theissue/templates/page.html]: Page template doesn't have a title.

**Review by madhusudhand — COMMENTED** [review]
Theme looks great!

I have posted few minor questions before shipping.

Can you also provide license information for the image used in the screenshot?

File: theissue/parts/post-meta.html (hunk 1 above)
Comment (henriqueiamarino) [reply theissue/parts/post-meta.html]: This is a symbol I expected to be present as a typeface glyph. Apparently, it's not.
So I think I'll remove it and update the theme. Is that okay?

Screenshot 2023-07-06 at 11 34 35

File: theissue/templates/page.html (hunk 2 above)
Comment (henriqueiamarino) [reply theissue/templates/page.html]: That is intended, no worries.

**Comment by madhusudhand** [comment]
@henriqueiamarino Thanks for addressing the issues. Renamed the theme from `theissue` to just `issue`.

Tested and it is now ready to 🚢

**Comment by henriqueiamarino** [comment]
Thanks @madhusudhand. I'll work on the showcase and the demo site then.

**Pull request merged**